pkg/ddl/notifier/subscribe.go

```diff
@@ -185,10 +177,17 @@ func (n *ddlNotifier) processEvents(ctx context.Context) error {
 			}
 		}
 
+		if intest.InTest {
+			if n.handlersBitMap == 0 {
```

Comment:
I don't really get it. Could you please explain the purpose of this change?

Reply:
I updated the code comment in https://github.com/pingcap/tidb/pull/56776/commits/fc91dd7aea27fb1c22c1c6121734840b922f720d